public_html/index.html

```diff
@@ -1,41 +1,68 @@
-<!DOCTYPE html>
-<html>
+
```

Comment:
Sorry Alistair, hopefully improved now.

On 6 July 2016 at 12:33, Alistair Chisholm <notifications@github.com> wrote:

> In public_html/index.html
> <https://github.com/gordon-qcode/hello-world/pull/4#discussion_r69714270>:
>
> > @@ -1,41 +1,68 @@
> > -<!DOCTYPE html>
> > -<html>
> > +
>
> The doctype declaration seems to be missing here
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/gordon-qcode/hello-world/pull/4/files/61c7678bee736d5c286e3165df0ba91828038283#r69714270>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AS-1f-g8E_eDXSbq23d8QqKr9xquPKa9ks5qS5JwgaJpZM4JF_QC>
> .
>